tools/run_tests/run_tests.py

```diff
@@ -191,19 +191,25 @@ def __init__(self):
       self._tests = json.load(f)
 
   def test_specs(self, config, travis):
-    modules = [config.job_spec(['tools/run_tests/run_python.sh', '-m',
```

Comment:
How much of these build-related configuration changes can be split out into a different changelist? Can we change the building and tooling code first and then add the code in a later change which makes a much small (something like "adding a single value to an array") tool change?